avaframe/com1DFA/com1DFA.py

```diff
@@ -1810,10 +1810,10 @@ def readPartFromPickle(inDir, flagAvaDir=False):
     """
 
     if flagAvaDir:
-        inDir = os.path.join(inDir, 'Outputs', 'com1DFA', 'particles')
+        inDir = pathlib.Path(inDir, 'Outputs', 'com1DFA', 'particles')
 
     # search for all pickles within directory
-    PartDicts = sorted(glob.glob(os.path.join(inDir, '*.p')))
+    PartDicts = sorted(inDir.glob('*.p'))
```

Comment:
sorted(inDir.glob('*.p')) do you need a list here? sorted(list(inDir.glob('*.p')))